Cython/Compiler/Symtab.py

```diff
@@ -752,7 +754,8 @@ def register_pyfunction(self, entry):
 
     def declare_cfunction(self, name, type, pos,
                           cname=None, visibility='private', api=0, in_pxd=0,
-                          defining=0, modifiers=(), utility_code=None, overridable=False):
+                          defining=0, modifiers=(), utility_code=None,
+                          overridable=False, is_property=False):
```

Comment:
Since the flag is unused here, maybe we should add an assertion that it's not set to true anywhere?
Or, actually, not have it here at all?

Reply:
In the test, `Scope.declare_cfunction` is called once from `CFuncDefNode.analyse_declarations` for `__pyx_unpickle_Foo__set_state`. Removing the kwarg means checking the `env` type before the call

Reply:
reworked to avoid adding the kwarg